olm-catalog/knative-eventing.v0.3.0.clusterserviceversion.yaml

```diff
@@ -64,7 +64,7 @@ spec:
         - apiGroups:
           - eventing.knative.dev
           resources:
-          - channels/finalizers
+          - '*/finalizers'
```

Comment:
Upstream problem as well?

Reply:
That's a very good question - I wonder if GKE doesn't run the OwnerReferencesPermissionEnforcement admission controller?

https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#ownerreferencespermissionenforcement

Reply:
Maybe? Not sure about this one. Possibly also related to the olm-hack?